README.md

```diff
@@ -1,14 +1,15 @@
-# [QOwnNotes](http://www.qownnotes.org "QOwnNotes Offical Site") 
+# [QOwnNotes](http://www.qownnotes.org "QOwnNotes Offical Site")
 
-[Installation](http://www.qownnotes.org/installation) | 
-[Changelog](https://github.com/pbek/QOwnNotes/blob/develop/CHANGELOG.md) | 
-[Issues](https://github.com/pbek/QOwnNotes/issues) | 
+[Installation](http://www.qownnotes.org/installation) |
+[Changelog](https://github.com/pbek/QOwnNotes/blob/develop/CHANGELOG.md) |
+[Issues](https://github.com/pbek/QOwnNotes/issues) |
 [Releases](https://github.com/pbek/QOwnNotes/releases) |
 [Shortcuts](https://github.com/pbek/QOwnNotes/blob/develop/SHORTCUTS.md) |
 [Documentation](http://docs.qownnotes.org) |
 [Knowledge base](http://www.qownnotes.org/Knowledge-base) |
 [Screenshots](http://www.qownnotes.org) |
-[ownCloud Apps page](https://apps.owncloud.com/content/show.php?content=168497) |
+[ownCloud Apps](https://marketplace.owncloud.com/) |
+[Nextcloud Apps](https://apps.nextcloud.com/) |
```

Comment:
why did you add the apps pages of oC and Nextcloud

Reply:
I added the Nc apps page because you had already the oC apps page: after updating the oC apps link (`marketplace` insead of `apps` in `owncloud.com`), I found missing a link to the Nc apps page.

Reply:
My link was a link to the app page of QOwnNotesAPI, I guess it would make more sense to add links to those, see https://github.com/pbek/qownnotesapi/blob/develop/README.md

Reply:
You're right, it makes more sense

Reply:
Do you care to add the links? :)